finetuner-workflow/finetune-download-dataset.yaml

```diff
@@ -5,9 +5,23 @@ metadata:
 spec:
   template:
     spec:
+      # The dataset downloader runs as the nonroot user so the dataset folder in the PVC
+      # needs the correct permissions.
+      initContainers:
+        - name: init
+          image: alpine:3.17
+          command: [ "/bin/sh" ]
+          args:
+            - -c
+            - >-
+              mkdir -p /mnt/pvc/dataset && 
+              chmod o+rw,g+s /mnt/pvc/dataset
```

Comment:
this should probably be templated as part of the argo job, as `dataset` is a parameter in the workflow.

Reply:
Consider making this part of the argo workflow.